src/tests/class_tests/openms/source/DBSuitability_test.cpp

```diff
@@ -49,9 +49,43 @@
 using namespace OpenMS;
 using namespace std;
 
+#include <OpenMS/FORMAT/FASTAFile.h>
+#include <OpenMS/FORMAT/MzMLFile.h>
+#include <OpenMS/FORMAT/IdXMLFile.h>
+#include <OpenMS/KERNEL/MSExperiment.h>
+
+
 START_TEST(Suitability, "$Id$")
 
-/////////////////////////////////////////////////////////////
+MSExperiment exp;
+MzMLFile m;
+m.load("C:\\Development\\TOPPAS_Outputs\\Corrected_Suitability_test_files\\TOPPAS_out\\006-FileConverter-out\\BSA1.mzML", exp);
```

Comment:
you need platform independent paths here.
Try `m.load(File::find("./examples/BSA/BSA1.mzML"));`